drivers/clock_control/clock_stm32_ll_common.h

```diff
@@ -30,10 +30,15 @@
 	#define MCO2_SOURCE		LL_RCC_MCO2SOURCE_PLLCLK
 #endif
 
+#ifdef CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL
 void config_pll_init(LL_UTILS_PLLInitTypeDef *pllinit);
+#endif /* CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL */
 void config_enable_default_clocks(void);
 
 /* Section for functions not available in every Cube packages */
 void LL_RCC_MSI_Disable(void);
 
+
+int stm32_clock_control_init(const struct device *dev);
```

Comment:
If made public it's worth adding some docstrings

Reply:
ok, adding some lines